proto/side/gmm/pool_asset.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package side.gmm;
+option go_package = "github.com/sideprotocol/side/x/gmm/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto"; 
+
+message PoolAsset {
+  cosmos.base.v1beta1.Coin token = 1 [(gogoproto.nullable) = false]; 
+  string weight = 2 [
```

Comment:
not all pool are weighted pool.

Reply:
yep. about other pool, it will set as 0 as a default. 
no need to setup

Reply:
in othercase, there should be `50:50`, so the value is `50`

Reply:
yep. it will change according to the frontend request. 
For example, if we want to create weight pool, we have to set this value from 0 to 100 values. 
so total sum is just 100. 
In other case, for example at stable pool, it has no meaning because it doesn't be used anywhere.